Add io.github.nokse22.teleprompter

Nokse22 opened the pull request:
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

I use `--filesystem=home:ro` because I open a file chooser dialog, but only to read files.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

Nokse22 commented:
bot, build io.github.nokse22.teleprompter

hfiguiere commented on io.github.nokse22.teleprompter.json:
```diff
@@ -0,0 +1,40 @@
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=home:ro"
```
this is a very broad permission. Given that it uses GNOME it should use the portals.

hfiguiere commented:
bot, build io.github.nokse22.teleprompter

Nokse22 commented:
bot, build io.github.nokse22.teleprompter

Nokse22 commented:
I removed the unnecessary permissions and fixed the other problems.
Can't see the screenshots on the Software app details page.

Everything else is working fine.

barthalion commented:
/merge
